superset-frontend/spec/javascripts/dashboard/components/gridComponents/Tabs_spec.jsx

```diff
@@ -53,16 +55,19 @@ describe('Tabs', () => {
     deleteComponent() {},
     updateComponents() {},
     logEvent() {},
+    setMountedTab() {},
   };
 
   function setup(overrideProps) {
     // We have to wrap provide DragDropContext for the underlying DragDroppable
     // otherwise we cannot assert on DragDroppable children
     const wrapper = mount(
       <Provider store={mockStoreWithTabs}>
-        <WithDragDropContext>
-          <Tabs {...props} {...overrideProps} />
-        </WithDragDropContext>
+        <ThemeProvider theme={supersetTheme}>
```

Comment:
If you'd rather not import the `supersetTheme` and `ThemeProvider` to use here, you can instead `import { styledMount as mount } from 'spec/helpers/theming'` and then just do 
```
<Provider store={mockStoreWithTabs}>
  <WithDragDropContext>
    <Tabs {...props} {...overrideProps} />
  </WithDragDropContext>
</Provider>
```